programs/sll_remove_dups.py

```diff
@@ -0,0 +1,48 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        self.head = None
+
+    def display(self):
```

Comment:
No need to implement this if you had `__str__` method implemented in `single_linked_lists`'s `Node` class as suggested.